### PR TITLE
Sync

### PR DESCRIPTION
**Pull request by wtrocki**
Branch contains all Sync related changes.

### CODE REVIEW

**Comment by laurafitzgerald** [comment]
@wtrocki  thanks for the pr. As I mentioned in the related [JIRA](https://issues.jboss.org/browse/AEROGEAR-7882) would it be better to merge this one and build it and review the content as part of the [review docs JIRA](https://issues.jboss.org/browse/AEROGEAR-7882) cc @jstaffor

**Comment by jhellar** [comment]
`modules/ROOT/pages/sync/authorization.adoc` should be included in `modules/ROOT/pages/data_sync.adoc` to be visible

**Comment by wtrocki** [comment]
@jhellar Alternatively we can join them together in single auth section

**Comment by wtrocki** [comment]
We can create fix PR's on top of this.

**Comment by wtrocki** [comment]
Reviewing this myself

File: modules/ROOT/pages/sync/ref_terminology.adoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
+:context: ref_terminology_{sync-service}
+
+[id='{context}_ref_terminology']
+
+= {sync-service} Terminology
+
+This section describes terminology that is associated with {sync-service}.
+
+GraphQL:: A query language for data that is more powerfull than a REST API. See link:https://graphql.org/learn[GraphQL]
```
Comment (ziccardi) [inline modules/ROOT/pages/sync/ref_terminology.adoc]: `powerfull` should be `powerful`

**Comment by wtrocki** [comment]
@ziccardi Agreed with your suggestions. Let's create PR with suggestions on top of that.

**Comment by danielpassos** [comment]
Sync branch should be rebased before merge to remove unnecessary _Merge pull request_ commits

**Comment by wtrocki** [comment]
@danielpassos Trying to rebase this now but it could be challenging :)

**Comment by wtrocki** [comment]
@danielpassos https://github.com/aerogear/mobile-docs/compare/sync-rebased?expand=1 is correct and correspond to this PR. Feel free to force push. 

Done most of the checks and I think we are ready to get that merged.

**Comment by finp** [comment]
preview at https://5b8fe9e173f2cf0c34c520c5--mobile-docs.netlify.com/aerogear/latest/data-sync/#ref_terminology_Data%20Sync_ref_terminology

**Comment by danielpassos** [comment]
@wtrocki Rebased!

**Comment by danielpassos** [comment]
@finp tabs does not work correctly on that link: https://5b8fe9e173f2cf0c34c520c5--mobile-docs.netlify.com/aerogear/latest/data-sync/#sdk

**Comment by finp** [comment]
@wtrocki  looks good, two things:

* data sync seems important, should we move it up the list of services, just below IDM?
* We need a note introducing the fact that it's 0.1.0 (new attribute)

**Comment by finp** [comment]
@danielpassos  yes, known issue for this type of preview (but hey, it's easy to do and share)
see readme at https://github.com/finp/mobile-netlify

File: modules/ROOT/pages/sync/data_sources_ui.adoc [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+=== Delete an Existing Data Source
+
+To delete a data source, choose the *Delete Data Source* option from the data source menu. Next confirm that you wish to delete the data source.
+
+NOTE: When you delete a data source, all resolvers that are attached to it will also be deleted.
+
+=== Create a New Data Source
+
+If no data source exists, a new data source can be created by clicking the *Add Data Source* button. Select the type of your data source from the dropdown.
+For more information about the _In Memory_ option have a look at xref:#in-memory-option[the In Memory Option]. Information about the Postgres option can be found xref:#in-memory-option[here].
```
Comment (ziccardi) [inline modules/ROOT/pages/sync/data_sources_ui.adoc]: This should point to postgres-option

**Comment by danielpassos** [comment]
@ziccardi Please remove the `Merge pull request` commit

**Comment by pmacko1** [comment]
Hi guys, is https://docs.aerogear.org/aerogear/latest/con_services.html updated in some pr? Thanks

**Comment by danielpassos** [comment]
@wtrocki @ziccardi I have removed the `Merge pull request` commits

**Pull request merged**